### ISSUE TITLE
language settings for en-GB and en-US are not recognized

### ISSUE THREAD

**Issue by eBlick** [terminal42/contao-notification_center#93]
I have multiple englisch speaking domains with individual country-codes like en-GB and en-US.
The nc does not recognize them and gets the fallback.
Only when selecting englisch-english it will be recognized.

![bildschirmfoto 2016-03-31 um 15 11 38](https://cloud.githubusercontent.com/assets/6641281/14176805/3e9cafc0-f753-11e5-8734-9897356e15bc.png)

**Comment by Toflar** [comment]
Can you check if https://github.com/terminal42/contao-notification_center/commit/91af3a03e8946aca8e466ebb7af94c49fca3fb75 fixes your issue?

**Comment by eBlick** [comment]
unfortunately not.
files changed, cache deleted, same result.

**Comment by eBlick** [comment]
there was a similar result with isotope caused by en-US and en_US marking if this helps...

**Comment by Toflar** [comment]
Which notification type are you using?

**Comment by eBlick** [comment]
form submission

**Comment by Toflar** [comment]
Wenn du aus dem `$GLOBALS['TL_LANGUAGE']` ein `str_replace('-', '_', $GLOBALS['TL_LANGUAGE'])` machst, dann funktioniert's? Sorry, ich kann es im Moment nirgendwo testen bzw. es fehlt mir grad die Zeit dazu. Ansonsten muss das Ticket einfach erst mal warten.

**Comment by eBlick** [comment]
sorry, no difference.
I've tried the hardcoded version, too:  'en-GB' and 'en_GB', no difference.
'en' - only value  works fine.

**Comment by qzminski** [comment]
Fixed in 80f535b.

**Comment by qzminski** [comment]
@eBlick can you test the 80f535b?

**Comment by eBlick** [comment]
I tested again with 80f535b. No positive result though.

Tested with en-GB. Fallback works.

**Comment by qzminski** [comment]
Just to make sure, you have set the ```en-GB``` in your website root language settings, and not ```en-gb```?

@Toflar do you also mind giving it a quick test shot?

**Comment by eBlick** [comment]
Yes I have:
![bildschirmfoto 2016-05-04 um 09 40 29](https://cloud.githubusercontent.com/assets/6641281/15007869/636be868-11dc-11e6-8db0-ea430edffecb.png)

The Changelanguage Module is based on this attribute and working well...

**Comment by Toflar** [comment]
I think it's the combination of both, my and Kamil's commit and you only checked for one ;) Will release.

**Comment by eBlick** [comment]
I'm rather confused now!
I have tested the en-GB version only and it worked!
No I have tested the en-US Version and no notification was generated.
I have tripple-checked all settings: 
site-configuration = en-US
notification settings see screenshot.

I also removed all insert tags in the notification text. (I think there used to be a problem when inserting them via drag&drop?)
![bildschirmfoto 2016-05-05 um 16 05 30](https://cloud.githubusercontent.com/assets/6641281/15045747/9635725e-12dc-11e6-8500-b6cdd9fa8424.png)
![bildschirmfoto 2016-05-05 um 16 18 13](https://cloud.githubusercontent.com/assets/6641281/15045868/29632292-12dd-11e6-8c29-a1785dddff15.png)
